### DISCUSSION TITLE
【寻人启事】光映广场的大灰狼失踪了！

### DISCUSSION THREAD

**Discussion by Lixia256** [share121/inter-knot#2547] [其他]
![photo_2024-08-17_15-41-49](https://github.com/user-attachments/assets/b23b36f9-b48b-41af-9b27-83763c5063bb)
继莱卡恩先生被数以万计的福瑞控疯狂折磨一个月以至于百日内无法下床后，照片中的先生成为了下一位男同福瑞控手中的受害者，本台紧急发布寻人启事，因为我们没还撸到他！

**Comment by jimmy123-debug** [comment]
这是新角色吗

**Reply by Lixia256** [reply]
光映广场外滩的NPC

**Comment by MikUranium** [comment]
我接下了！

**Comment by beta2778** [comment]
操他橘瓣

**Reply by Lixia256** [reply]
吓死了大姐
